<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Packet Eye</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 패킷 목록 QListWidget -->
   <widget class="QListWidget" name="packetListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1000</width>
      <height>400</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>패킷 목록</string>
    </property>
   </widget>
   <!-- 네트워크 장치 QListWidget -->
   <widget class="QListWidget" name="deviceListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>600</width>
      <height>120</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>네트워크 장치 목록</string>
    </property>
   </widget>
   <!-- 목록 불러오기 QPushButton -->
   <widget class="QPushButton" name="captureButton">
    <property name="geometry">
     <rect>
      <x>
      1100</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>목록 불러오기</string>
    </property>
   </widget>
   <!-- 캡처 시작 QPushButton -->
   <widget class="QPushButton" name="startCaptureButton">
    <property name="geometry">
     <rect>
      <x>
      1100</x>
      <y>80</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>캡처 시작</string>
    </property>
   </widget>
   <!-- 캡처 중지 QPushButton -->
   <widget class="QPushButton" name="stopCaptureButton">
    <property name="geometry">
     <rect>
      <x>
      1100</x>
      <y>140</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>캡처 중지</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>